Modify framework_objc module to include ringrtc

When building for iOS, the build_ios_libs.py script is used which calls
gn for the framework_objc target. We modify the BUILD.gn configuration
for framework_objc so that it refences the following RingRTC targets as
dependencies:

src/rffi/BUILD.gn
This ends up being a static library linked in with the resulting
WebRTC.framework.
---
 sdk/BUILD.gn |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/sdk/BUILD.gn b/sdk/BUILD.gn
index 2c8a114c27ca..a22bf70524b6 100644
--- a/sdk/BUILD.gn
+++ b/sdk/BUILD.gn
@@ -10,6 +10,7 @@ import("../webrtc.gni")
 if (is_ios) {
   import("//build/config/ios/ios_sdk.gni")
   import("//build/config/ios/rules.gni")
+  import("//ringrtc/ringrtc-src-dir.gni")
 }
 if (is_mac) {
   import("//build/config/mac/rules.gni")
@@ -1347,6 +1348,7 @@ if (is_ios || is_mac) {
           ":videocodec_objc",
           ":videotoolbox_objc",
           "../rtc_base:rtc_base_approved",
+          "${ringrtc_src}/rffi:libringrtc_rffi",
         ]
         if (rtc_use_metal_rendering) {
           deps += [ ":metal_objc" ]
